staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -285,7 +285,13 @@ func (c *AvailableConditionController) sync(key string) error {
 					results <- err
 					return
 				}
-				discoveryURL.Path = "/apis/" + apiService.Spec.Group + "/" + apiService.Spec.Version
+				// skip legacyAPIService health check when it is delegated to the core apiserver
```

Comment:
This comment is confusing-- we already know the service isn't local due to line 278? So how can this be delegated to the core apiserver?

Reply:
Changed the comment to `skip legacyAPIService health check when it is delegated to a service`. Hope it better describe the intention.